content/components/page-header.mdx

```diff
@@ -304,6 +304,8 @@ The title bar can include a static page title, actions, and leading and trailing
 Pageheader allows usage of different HTML landmarks to create a more accessible UI.
 The usage of `header` and `nav` regions are allowed within PageHeader, both with different use cases.
 
+If the `PageHeader` is not contained within or contains a landmark, consider using the `role` prop to indicate one and provide it with an accessible name (perhaps the same as `PageHeader.Title`) via `aria-label`.
```

Comment:
```suggestion
If the `PageHeader` is not contained within a landmark, consider using the `role` prop to indicate one and provide it with an accessible name (perhaps the same as `PageHeader.Title`) via `aria-label`.
```

Reply:
I agree this wording is kind of weird, I'm trying to guide users to prevent a "Banner landmark cannot be nested within another landmark." error , which means there shouldn't be a landmark up in the chain or within the chain, but maybe we do not need to get that deep into it? 🤔